src/ext/java/org/opentripplanner/ext/flex/FlexTripsMapper.java

```diff
@@ -34,10 +35,15 @@ public class FlexTripsMapper {
     for (Trip trip : stopTimesByTrip.keys()) {
       /* Fetch the stop times for this trip. Copy the list since it's immutable. */
       List<StopTime> stopTimes = new ArrayList<>(stopTimesByTrip.get(trip));
-
       if (UnscheduledTrip.isUnscheduledTrip(stopTimes)) {
+        var modifier = builder.getFlexDurationFactors().getOrDefault(trip, TimePenalty.ZERO);
```

Comment:
Should we stick to call to calling them modifiers instead of factors? Or is there some difference?

Reply:
Yeah, sorry, I did several renames of this field and made a mess. Lets discuss in the dev meeting what this field should be called.

Reply:
Lets use timePenalty here - modifier is too wide. Factor is not correct, it is more than a factor.